spring-boot/dependency-injection/src/test/java/com/example/test/constructorinjection/TestCakeClassConstructorInjection.java

```diff
@@ -0,0 +1,25 @@
+package com.example.test.constructorinjection;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.example.ExampleApplicationCI;
+import com.example.constructorinjection.Cake;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = ExampleApplicationCI.class)
+public class TestCakeClassConstructorInjection {
+
+	@Autowired
+	Cake cake;
+	
+	@Test
+	public void testConstructorInjection() {	
+		String testColor=cake.getFlavor().getColor();
```

Comment:
Can you run a formatter over all the code, please? Sometimes there's spaces and sometimes not. The default formatter of whatever IDE you're using is fine, just that it's consistent.